src/main/java/com/savvato/tribeapp/entities/ReviewDecision.java

```diff
@@ -0,0 +1,54 @@
+package com.savvato.tribeapp.entities;
+
+import javax.persistence.*;
+import java.util.Calendar;
+
+@Entity
+@IdClass(ReviewDecisionId.class)
+public class ReviewDecision {
+    @Id
+    public Long reviewId;
+    @Id
+    public Long userId;
+    public Long reasonId;
+    private java.sql.Timestamp created;
+
+
+    public ReviewDecision(Long reviewId, Long userId, Long reasonId) {
+        this.reviewId = reviewId;
+        this.userId = userId;
+        this.reasonId = reasonId;
+        setCreated();
+    }
+
```

Comment:
Add a default (no-parameter) constructor, too.

Reply:
I have made this change now.